tests/integration/test_dashboards.py

```diff
@@ -1,12 +1,57 @@
 from pathlib import Path
 
-from databricks.sdk import WorkspaceClient
+import pytest
 
-from databricks.labs.lsql.dashboards import Dashboards
+from databricks.labs.lsql.dashboard import Dashboard
+from databricks.labs.lsql.lakeview.model import CounterSpec
+
+
+@pytest.fixture
+def dashboard_id(ws, make_random):
```

Comment:
I am missing a "list" endpoint on the lakeview API. Could we request one?

Reply:
they are files...

Reply:
Are suggesting to list all files in the Workspace to mimic a list behavior?

Reply:
only `.lvdash.json`